scripts/abins/kpointsdata.py

```diff
@@ -177,3 +154,30 @@ def __getitem__(self, item):  # noqa F811
             return [self[i] for i in range(len(self))[item]]
 
         return self._data[item]
+
+    class JSONableData(TypedDict):
```

Comment:
whats the difference between this and https://github.com/mantidproject/mantid/pull/35532/files#diff-10cd79b62f0cea50843f4972100d059d8fd47ebb77a7163edcd7c14e0d8572b3R79 

Unless I am mistaken the name is the same.

Reply:
They are different data structures, used to give more robust type-hinting than a bare dictionary. Because they are defined inside the class they have different namespaces. For all practical purposes, one of them is named `AbinsData.JSONableData` and the other one is named `KpointsData.JSONableData`.